noir-projects/noir-contracts/contracts/test/spam_contract/src/types/balance_set.nr

```diff
@@ -1,57 +1,64 @@
-// This file is copied from the token contract.
```

Comment:
The change here is large because this was a copy from the token contract but at some older time. I couldn't be bothered updating multiple versions of the same thing so I just copied the `balance_set.nr` from the token contract here.